### DISCUSSION TITLE
make install without_doc

### DISCUSSION THREAD

**Discussion by mickae1** [FreeTDS/freetds#434] [Ideas]
Hi,

Could you add the possibility to install without the documentation ?

Thanks,

**Comment by fziglio** [comment]
It's not hard, mainly adding a configure option to disable documentation directory should be enough. However this should not be a bit issue if you use standard packaging systems, usually packagers install all and split the installed files or strip/move what they don't like.

What's the exact issue you are trying to solve and what's the environment?

**Comment by taozuhong** [comment]
OS: Fedora 38
freetds: 1.4.10

freetds can't build due to the doc issue, the detail as follow:
```
make[2]: Leaving directory '/home/taozuhong/freetds-1.4.10/src'
make[1]: Leaving directory '/home/taozuhong/freetds-1.4.10/src'
Making all in doc
make[1]: Entering directory '/home/taozuhong/freetds-1.4.10/doc'
git log -1 '--pretty=format:<!-- Autogenerated -->%n<!ENTITY ug.date "Date: %ai">%n<!ENTITY ug.id "Git Id: %H">' HEAD -- userguide.xml > userguide_desc.xml~
fatal: not a git repository (or any of the parent directories): .git
make[1]: *** [Makefile:770: userguide_desc.xml] Error 128
make[1]: Leaving directory '/home/taozuhong/freetds-1.4.10/doc'
make: *** [Makefile:464: all-recursive] Error 1
```

**Reply by freddy77** [reply]
It works for me, all necessary files should be in the distribution. Probably you changed something that require to build from sources, given the command documentation (like `userguide.xml`).

I just tried even with a clean docker Fedora 38 with no issue.

**Reply by taozuhong** [reply]
I'm make sure that I didn't change anything, just unpack and re-make it, the output as follow:
```
make[1]: Leaving directory '/home/andytao/freetds-1.4.10/src'
Making all in doc
make[1]: Entering directory '/home/andytao/freetds-1.4.10/doc'
if test -h reference; then rm reference; fi
/usr/bin/mkdir -p ./reference 2>&1
cd .. && make doxy 2> doc/doxy.log
make[2]: Entering directory '/home/andytao/freetds-1.4.10'
/usr/bin/mkdir -p doc/reference
rm -rf doc/reference/*
SRCDIR=. VERSION=1.4.10 doxygen ./tds.dox
if test -d doc/reference/html ; then mv doc/reference/html/* doc/reference; rmdir doc/reference/html; fi
make[2]: Leaving directory '/home/andytao/freetds-1.4.10'
if test -r ././reference/index.html -a ! reference/index.html ; then \
        cp -R ././reference .; \
fi
if test ! -r reference/index.html ; then \
        cd ./reference && \
        echo '<html> <P>at <a HREF="http://www.freetds.org/reference/">www.freetds.org</a> </html>' > .index.html &&  \
        mv .index.html index.html; \
fi
git log -1 '--pretty=format:<!-- Autogenerated -->%n<!ENTITY ug.date "Date: %ai">%n<!ENTITY ug.id "Git Id: %H">' HEAD -- userguide.xml > userguide_desc.xml~
fatal: not a git repository (or any of the parent directories): .git
make[1]: *** [Makefile:770: userguide_desc.xml] Error 128
make[1]: Leaving directory '/home/andytao/freetds-1.4.10/doc'
make: *** [Makefile:464: all-recursive] Error 1
```

**Comment by freddy77** [comment]
Oh... now I understand. You downloaded "Source file" assets from https://github.com/FreeTDS/freetds/releases/tag/v1.4.10 (or similar). Yes, that files are not complete. Not sure how to delete these automatic GH packages which are wrong! I'll have to look at GH documentation hoping there's a way to remove them.

Good catch!

**Comment by freddy77** [comment]
Arghh... https://github.com/orgs/community/discussions/6003 :disappointed:

**Comment by freddy77** [comment]
On newer releases I'm adding a comment telling to not use **Source code** files
